Documentation/policy/language.rst

```diff
@@ -441,34 +441,47 @@ but not CIDR prefix ``10.96.0.0/12``
 DNS based
 ---------
 
-``toFQDNs`` simplifies specifying egress policy to IPs of remote, external,
-peers. The DNS lookup for each ``matchName`` is done periodically by
-``cilium-agent`` and the result is used to regenerate endpoint policy. This
-allows tracking changing IPs or sets of IPs that may not be known a priori.
-Despite the naming, the ``matchName`` field does not have to be a
-fully-qualified domain name. In cases where search domains are configured, the
-DNS lookups from ``cilium`` will not be qualified and will utilize the search
-list.
+``toFQDNs`` simplifies specifying Layer 3 egress policy to IPs of remote,
+external, peers. It allows tracking changing IPs that may not be known a
```

Comment:
This paragraph needs an introduction on what DNS based actually means. It's great to provide the implementation details as well but before that, an introduction must provide the purpose of DNS based policies.

Reply:
I tried doing what `CIDR based` did, let me know if this is what you had in mind. I feel I still went into implementation details too soon, and maybe I explained the use cases in an unclear way? (I don't know if I explained what it means :/ )